oauthlib/common.py

```diff
@@ -44,6 +44,7 @@
 SANITIZE_PATTERN = re.compile(r'([^&;]*(?:password|token)[^=]*=)[^&;]+', re.IGNORECASE)
 INVALID_HEX_PATTERN = re.compile(r'%[^0-9A-Fa-f]|%[0-9A-Fa-f][^0-9A-Fa-f]')
 
+TOKEN_LENGTH = 40
```

Comment:
Same case: `NAUTILUS_TOKEN_LENGTH = getattr(settings, 'OAUTH2_TOKEN_LENGTH' or something similar, 30)`, it will prevent default behaviour.

Reply:
This doesn't seem right. oauthlib has nothing to do with Django settings - it is generic package providing Oauth support.